core/src/test/scala/unit/kafka/server/ControllerApisTest.scala

```diff
@@ -113,12 +113,8 @@ class ControllerApisTest {
 
     val request = buildRequest(brokerRegistrationRequest)
 
-    val capturedResponse: Capture[Option[AbstractResponse]] = EasyMock.newCapture()
```

Comment:
These and other captures had to change since we can only easily mock RequestChannel. The three argument `sendResponse` was moved to the helper class.